test/migrations-test.coffee

```diff
@@ -138,4 +138,4 @@ describe "Migrations",  ->
 
     describe "v-1.19.0 changes", ->
       it "should have complexity setting", ->
-        @result.settings.complexity.should.equal "basic"
+        @result.settings.complexity.should.equal 1
```

Comment:
Everywhere else you have used the more literate style eg `AppSettingsStore.Complexity.basic`.  Is there a reason not to do that here?

Reply:
Elsewhere, everything can reference the same the same value for `AppSettingsStore.Complexity.basic` and stay internally consistent, so if we change it under the hood, nothing at all changes (well, except for the numeric comparisons, now). However, the saved data exists independently of that, and will always be `1` once set that way. It seemed that making this explicit would help us avoid errors if, say, we change the value of `AppSettingsStore.Complexity.basic` and just forget that old data will still be using `1`. Does that make sense?

Reply:
If we keep it `AppSettingsStore.Complexity.basic` a failing test will remind us to handle a migration to update the number?  Otherwise, I see your reasoning, and  I am quite fine with ether.

Reply:
I guess it's a little odd, because the migration itself uses the property key. But I just feel like the migrations tests should test the raw data, not our interpretations of the data, because that will be the literal result of the migration.